upgrade/1.4.2/README.md

```diff
@@ -174,6 +174,43 @@ This step will install the `smart-mon` rpm on storage nodes, and reconfigure the
    /usr/share/doc/csm/scripts/operations/ceph/enable-smart-mon-storage-nodes.sh
    ```
 
+## Configure NCN nodes without restart
```

Comment:
This should be added to the table of contents at the top of the page.

Reply:
Done.